_pages/OutsideRange.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: page
+title: OutsizeRange MACRO
```

Comment:
Typo - also I thing he macros are snake cased and not camel cased. Probaly we should keep it consistent with the code.

Reply:
Fixed